include/deal.II/trilinos/nox.templates.h

```diff
@@ -70,7 +70,7 @@ namespace TrilinosWrappers
         }
 
         /**
-         * Initialize every element of this vector with gamma.
+         * Initialize every element of this vector with @p gamma.
```

Comment:
Given the change you made above, isn't `@p gamma.` also rendering `.` with the `tt` font? (Not that it matters much.)

Reply:
Yes, I think that you're right. I've fixed this.